master/pkg/tasks/task_gc.go

```diff
@@ -83,8 +83,10 @@ func (g GCCkptSpec) ToTaskSpec() TaskSpec {
 		strconv.Itoa(g.ExperimentID),
 		"--storage-config",
 		"/run/determined/checkpoint_gc/storage_config.json",
-		"--delete",
-		g.ToDelete,
+	}
+	if g.ToDelete != "" {
+		res.Entrypoint = append(res.Entrypoint, "--delete")
+		res.Entrypoint = append(res.Entrypoint, g.ToDelete)
```

Comment:
go style nit, just `res.Entrypoint = append(res.Entrypoint, "--delete", g.ToDelete`. but too late now.

Reply:
Good to know -- and thanks for the reviews!